osier/equations.py

```diff
@@ -250,6 +252,9 @@ def volatility(technology_list,
     solved_dispatch_model : :class:`osier.DispatchModel`
         A _solved_ dispatch model (i.e. with model results and objective
         attributes).
+    attribute : str
+        Indicates whether to calculate the volatility of electricity demand
+        or electricity price. Default is 'demand'.
```

Comment:
```suggestion
    attribute : {'demand', 'volatility'}
        Indicates whether to calculate the volatility of electricity demand
        or electricity price.
```
Don't need to specify the default in the docstring since it's in the function signature.